src/components/radio-button-group/radio-button-group.tsx

```diff
@@ -126,6 +143,20 @@ export class RadioButtonGroup {
     }
   };
 
+  private getFocusableRadioButton(): HTMLCalciteRadioButtonElement | null {
```

Comment:
Could you use the `focusFirstTabbable` helper in the dom utils?

This is what the modal does...

```tsx
 @Method()
  async setFocus(): Promise<void> {
    await componentLoaded(this);
    focusFirstTabbable(this.el);
  }
```

Would we want only 1 radio button within a group tabbable at a time? Seems like that might make for a good enhancement in the future.

Reply:
tried implementing `focusFirstTabbable` helper here,  it works well if none if the radio-button's in the group are disabled. The reason for this is the way we assign tabIndex at the `radio-button` level. The `tabIndex=0` only if the radio-button is `checked` or if its the first element in the group. I think this is a bug. Wonder if the assignment of `tabIndex` should stay at the group level not the `radio-button` level. 


> Would we want only 1 radio button within a group tabbable at a time? Seems like that might make for a good enhancement in the future.

The above is following w3 radio group pattern keyboard specs i believe. One button is tabbable and the user can use Arrow Keys to navigate between the items.